OsmAnd/src/net/osmand/plus/views/mapwidgets/configure/buttons/CompassButtonState.java

```diff
@@ -135,4 +136,12 @@ public Drawable getIcon(@DrawableRes int iconId, @ColorInt int color, boolean ni
 		}
 		return drawable;
 	}
+
+	@Nullable
+	@Override
+	public ButtonPositionSize getButtonPositionSize() {
```

Comment:
1) non-nullable
2) if it's get it should be saved, otherwise it's create. On get it returns incorrect state